be/src/column/chunk.cpp

```diff
@@ -197,6 +197,15 @@ void Chunk::append_selective(const Chunk& src, const uint32_t* indexes, uint32_t
     }
 }
 
+void Chunk::rolling_append_selective(Chunk& src, const uint32_t* indexes, uint32_t from, uint32_t size) {
+    DCHECK_EQ(_columns.size(), src.columns().size());
+    for (size_t i = 0; i < _columns.size(); ++i) {
```

Comment:
Nit: better to assign _column.size() to a local variable like num_columns

Reply:
Fixed